app/Helpers/util/mail.php

```diff
@@ -457,3 +457,45 @@ function sendSetRevisionEmail(
 
     return mail_utf8($email, $emailTitle, $msg);
 }
+
+/**
+ * Sends an email to a user informing them that their display name was successfully changed.
+ */
+function sendDisplayNameChangeConfirmationEmail(
+    User $user,
+    string $newDisplayName,
+): bool {
+    $emailTitle = "Username Change Approved";
+    $profileLink = "<a href='" . route('user.show', ['user' => $newDisplayName]) . "'>here</a>";
+
+    $msg = "Hello,<br><br>" .
+        "Great news! Your username change request to {$newDisplayName} has been approved.<br><br>" .
+
+        "You can now use your new username to log in everywhere on RetroAchievements.org.<br><br>" .
+
+        "Check out your updated profile {$profileLink}.<br><br>" .
+
+        "-- Your friends at RetroAchievements.org<br>";
+
+    return mail_utf8($user->EmailAddress, $emailTitle, $msg);
+}
+
+/**
+ * Sends an email to a user informing them that their display name change request was declined.
+ */
+function sendDisplayNameChangeDeclineEmail(
+    User $user,
+    string $desiredDisplayName,
+): bool {
+    $emailTitle = "About Your Username Change Request";
+
+    $msg = "Hello,<br><br>" .
+        "We've reviewed your request to change your username to {$desiredDisplayName}. " .
+        "Unfortunately we weren't able to approve this change at this time.<br><br>" .
+
+        "You're welcome to submit another request after a 30 day cooldown period has ended.<br><br> " .
```

Comment:
> Unfortunately we weren't able to approve this change at this time.

I'm struggling with the wording here. I'm not sure if it's the double "this" or the "weren't able" that is tripping me up. I think "We have reviewed your request to change your username to [name], and have decided not to approve it at this time." reads better.

Also note that contractions are less formal, and this feels like it has some level of formality. Suggest "We've reviewed" -> "We have reviewed" and "You're welcome" to "You are welcome".

Reply:
No problem - I have made these adjustments in latest.